### PR TITLE
created visualisation module and pi generator

### PR DESCRIPTION
**Pull request by ThatAmuzak**

### CODE REVIEW

File: StatChecker.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import FinalRNGen as rng
+import matplotlib.pyplot as plt
+
+random_nums_list=rng.ProduceRandomNumsList()
```
Comment (Tanny1810) [inline StatChecker.py]: give a range of -20 to 30

File: FinalRNGen.py [hunk 2]
```diff
@@ -36,21 +35,20 @@ def pseudo_rand_gen(low, high, size, seed):
 I will get into this after the code 
 '''
 
-ls = [69]  # completely pointless list, just to access a unique memory address each time the code is run
-# we now get the seed from the memory address of the list
-completely_random_seed = id(ls)
+def ProduceRandomNumsList(low=0, high=20, size=10000000):
```
Comment (Tanny1810) [inline FinalRNGen.py]: reduce default size to 100000

**Review by Tanny1810 — CHANGES_REQUESTED** [review]

File: StatChecker.py (hunk 1 above)
Comment (ThatAmuzak) [reply StatChecker.py]: changed the xlim for the plot as well

File: FinalRNGen.py (hunk 2 above)
Comment (ThatAmuzak) [reply FinalRNGen.py]: done

**Comment by Tanny1810** [comment]
Looks good GG!!

**Pull request merged**